omnibus/config/software/datadog-agent-integrations-py3.rb

```diff
@@ -175,7 +175,7 @@
 
     # We need to explicitly specify RUSTFLAGS for libssl and libcrypto
     # See https://github.com/pyca/cryptography/issues/8614#issuecomment-1489366475
-    if redhat? && !arm?
+    if redhat? && !arm_target?
```

Comment:
```suggestion
    if redhat_target? && !arm_target?
```

Reply:
I can't explain why I rolled this one back to `redhat` instead of `redhat_target`, so I'll move it back for the time being, and will update in the suse arm64 PR if required

Reply:
for the record, it seems that it wasn't required after all, or that something changed so that it's not required anymore: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/358171988